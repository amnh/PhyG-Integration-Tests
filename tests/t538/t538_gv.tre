digraph G {
	rankdir = LR;	node [ shape = none];

    0 [label=AlbanianTosk];
    1 [label=AncientIonicGreek];
    2 [label=ArmenianWestern];
    3 [label=Assamese];
    4 [label=Cornish];
    5 [label=Danish];
    6 [label=EasternFrisianSaterland];
    7 [label=Faroese];
    8 [label=Icelandic];
    9 [label=Irish];
    10 [label=KolschColognian];
    11 [label=Manx];
    12 [label=OldPrussian];
    13 [label=SanskritVedic];
    14 [label=Scots];
    15 [label=Yiddish];
    16 [label=HTU16];
    17 [label=HTU17];
    18 [label=HTU18];
    19 [label=HTU19];
    20 [label=HTU20];
    21 [label=HTU21];
    22 [label=HTU22];
    23 [label=HTU23];
    24 [label=HTU24];
    25 [label=HTU25];
    26 [label=HTU26];
    27 [label=HTU27];
    28 [label=HTU28];
    29 [label=HTU29];
    30 [label=HTU30];
    31 [label=HTU31];
    32 [label=HTU32];
    33 [label=HTU33];
    34 [label=HTU34];
    35 [label=HTU35];
    36 [label=HTU36];
    16 -> 0 [label=21.5];
    16 -> 29 [label=22.5];
    17 -> 11 [label=22.0];
    17 -> 35 [label=15.0];
    18 -> 1 [label=41.0];
    18 -> 9 [label=33.0];
    19 -> 13 [label=17.5];
    19 -> 36 [label=1.5];
    20 -> 21 [label=11.5];
    20 -> 31 [label=18.0];
    21 -> 23 [label=15.5];
    21 -> 24 [label=6.0];
    22 -> 5 [label=33.5];
    22 -> 12 [label=24.5];
    23 -> 25 [label=16.0];
    23 -> 27 [label=9.0];
    24 -> 7 [label=27.5];
    24 -> 34 [label=1.0];
    25 -> 17 [label=22.5];
    25 -> 26 [label=16.0];
    26 -> 6 [label=26.5];
    26 -> 28 [label=24.5];
    27 -> 14 [label=37.0];
    27 -> 32 [label=1.0];
    28 -> 15 [label=27.0];
    28 -> 33 [label=25.5];
    29 -> 20 [label=11.5];
    29 -> 22 [label=21.0];
    30 -> 19 [label=6.0];
    30 -> 32 [label=3.0];
    31 -> 2 [label=24.5];
    31 -> 30 [label=27.0];
    32 -> 3 [label=22.0];
    33 -> 10 [label=34.5];
    33 -> 34 [label=5.0];
    34 -> 8 [label=51.0];
    35 -> 4 [label=24.0];
    35 -> 36 [label=7.0];
    36 -> 18 [label=22.0];
}
//706.0
