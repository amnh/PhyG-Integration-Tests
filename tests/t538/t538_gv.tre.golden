digraph G {
	rankdir = LR;	edge [colorscheme=spectral11];	node [shape = none];

    0 [label=AlbanianTosk];
    1 [label=AncientIonicGreek];
    2 [label=ArmenianWestern];
    3 [label=Assamese];
    4 [label=Cornish];
    5 [label=Danish];
    6 [label=EasternFrisianSaterland];
    7 [label=Faroese];
    8 [label=Icelandic];
    9 [label=Irish];
    10 [label=KolschColognian];
    11 [label=Manx];
    12 [label=OldPrussian];
    13 [label=SanskritVedic];
    14 [label=Scots];
    15 [label=Yiddish];
    16 [label=HTU16];
    17 [label=HTU17];
    18 [label=HTU18];
    19 [label=HTU19];
    20 [label=HTU20];
    21 [label=HTU21];
    22 [label=HTU22];
    23 [label=HTU23];
    24 [label=HTU24];
    25 [label=HTU25];
    26 [label=HTU26];
    27 [label=HTU27];
    28 [label=HTU28];
    29 [label=HTU29];
    30 [label=HTU30];
    31 [label=HTU31];
    32 [label=HTU32];
    33 [label=HTU33];
    34 [label=HTU34];
    35 [label=HTU35];
    36 [label=HTU36];
    16 -> 0 [label=0.0];
    16 -> 25 [label=0.0];
    17 -> 1 [label=37.0];
    17 -> 28 [label=0.0];
    18 -> 7 [label=17.0];
    18 -> 19 [label=6.0];
    19 -> 31 [label=5.0];
    19 -> 35 [label=5.0];
    20 -> 15 [label=31.0];
    20 -> 18 [label=8.0];
    21 -> 2 [label=20.0];
    21 -> 22 [label=18.0];
    22 -> 13 [label=12.0];
    22 -> 23 [label=1.0];
    23 -> 26 [label=1.0];
    23 -> 36 [label=0.0];
    24 -> 5 [label=26.0];
    24 -> 12 [label=23.0];
    25 -> 8 [label=27.0];
    25 -> 29 [label=0.0];
    26 -> 3 [label=16.0];
    26 -> 32 [label=0.0];
    27 -> 4 [label=22.0];
    27 -> 11 [label=17.0];
    28 -> 9 [label=31.0];
    28 -> 34 [label=0.0];
    29 -> 10 [label=27.0];
    29 -> 30 [label=7.0];
    30 -> 20 [label=8.0];
    30 -> 21 [label=9.0];
    31 -> 6 [label=21.0];
    31 -> 32 [label=0.0];
    32 -> 14 [label=30.0];
    33 -> 24 [label=8.0];
    33 -> 34 [label=0.0];
    34 -> 27 [label=10.0];
    35 -> 17 [label=5.0];
    35 -> 36 [label=0.0];
    36 -> 33 [label=7.0];
}
//691.0
